bin/rails_next

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+export RAILS_NEXT=1
+bash rails "$@"
```

Comment:
This _seems_ to do what we would want from a `bin/rails_next` helper to pass through args to a rails 6 binary.